tests/phpunit/tests/rest-api/rest-global-styles-revisions-controller.php

```diff
@@ -380,413 +380,6 @@ public function test_get_item_schema() {
 		$this->assertArrayHasKey( 'modified_gmt', $properties, 'Schema properties array has "modified_gmt" key.' );
 	}
 
-	/**
```

Comment:
@ramonjd Removing tests doesn't seem like a good move. Did these tests run and pass before? If so, let's leave them.

Reply:
Yes they were all running and passing. I removed them because they were clones of the `WP_REST_Revisions_Controller` tests, so I thought it was duplication.

No worries though, I'll put them back in. 

Thanks for the feedback!

Reply:
Done!